package.json

```diff
@@ -24,7 +25,6 @@
   "homepage": "https://github.com/invault-team/frontend#readme",
   "devDependencies": {
     "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
```

Comment:
i thought class properties were necessary to maintain the state as a dynamic attribute inside the class

Reply:
@gugutz I prefer to use a more explicit aproach, by setting the state in the constructor and bind the necessary methods. 

```js
  constructor(props) {
    super(props)
    this.state = {currentStep: 0}
    this.handleNext = this.handleNext.bind(this)
    this.handleBack = this.handleBack.bind(this)
  }
```